backend/utils/db.py

```diff
@@ -166,12 +171,18 @@ async def update(
             RequestException: Unable to connect to zc_core
         """
 
+        # to ensure only either one of raw_query or data is sent
+        if data and raw_query:
+            raise Exception("Either one of data or raw_query is expected")
```

Comment:
Is there a reason this conditional is needed? what happens when we have both data and raw_query?

Reply:
![image](https://user-images.githubusercontent.com/82163647/216303211-feac7843-7293-41d7-b552-1f00cc6acb20.png)
this code snippet is from [ zc_core's `handlePut` function](https://github.com/zurichat/zc_core/blob/dev/data/write_data.go)

Because of this if-else block, when both data and raw_query are passed, zc_core will always choose to update with the raw_query. So if you intend to update a message for instance, you should only pass in data (a.k.a payload). But if you intend to update a thread message and you pass in both, zc_core will only update with raw_query anyways.

Reply:
By the way I've improved the if statement because I realised it wasn't accounting for a situation where both data and raw_query aren't provided.

Reply:
Okay sounds good.